modules/combined/tests/elastic_thermal_patch/elastic_thermal_jacobian_rz_smp_test.i

```diff
@@ -164,88 +150,81 @@
     boundary = 10
     function = temp
   [../]
-
-[] # BCs
+[]
 
 [Materials]
-
-  [./stiffStuff1]
-    type = Elastic
-    block = 1
-
-    disp_r = disp_x
-    disp_z = disp_y
-
+  [./elasticity_tensor]
+    type = ComputeIsotropicElasticityTensor
     youngs_modulus = 1e6
     poissons_ratio = 0.25
+  [../]
 
-    temp = temp
-    thermal_expansion = 1e-6
+  [./strain]
+    type = ComputeAxisymmetricRZIncrementalStrain
+    displacements = 'disp_x disp_y'
+    eigenstrain_names = eigenstrain
+  [../]
+
+  [./thermal_strain]
+    type = ComputeThermalExpansionEigenstrain
+    temperature = temp
+    stress_free_temperature = 117.56
+    thermal_expansion_coeff = 1e-6
+    incremental_form = true
+    eigenstrain_name = eigenstrain
+  [../]
+
+  [./stress]
+    type = ComputeStrainIncrementBasedStress
   [../]
 
   [./heat]
     type = HeatConductionMaterial
-    block = 1
-
     specific_heat = 0.116
     thermal_conductivity = 4.85e-4
   [../]
 
   [./density]
     type = Density
-    block = 1
     density = 0.283
     disp_r = disp_x
     disp_z = disp_y
   [../]
-
-[] # Materials
+[]
 
 [Preconditioning]
-#  [./FDP]
-#    type = FDP
-#    full = true
-#  [../]
   [./SMP]
     type = SMP
     full = true
   [../]
 []
 
 [Executioner]
-
   type = Transient
 
   #Preconditioned JFNK (default)
```

Comment:
can we get rid of this boilerplate comment it is in 90% of our input files and is totally useless